packages/core/src/error-reporter.ts

```diff
@@ -142,6 +147,7 @@ export class ErrorReporter {
 	}
 
 	error(e: unknown, options?: ReportingOptions) {
+		if (e instanceof ExecutionCancelledError) return;
```

Comment:
Cancelling an execution results in not one but two `error` logs with full stack traces, see [here](https://share.cleanshot.com/NLK1t7GK). 

`ExecutionCancelledError` is not an actual error to handle and the cancellation is already being reported normally via debug logs, so this change is to remove that noise.